### PR TITLE
Fixes #205

### LINKED ISSUE #205: Redesign home page
We've added new content and are adding more new content to the home page. I want to revisit the layout of this page, the order of content and overall hierarchy. 

The update that I'll be pushing through for Web Summit is **not** the new design.

### PR DESCRIPTION
**Pull request by ststimac**
Refine home page design with the addition of new content highlighting features

I did some text clean up in addition to what @alrra provided yesterday. Commas can be weird in English so adjust a few of those and reworded some things.

Home page looks much better IMO. Cleaned up spacing a bit so it's consistent between sections and I feel better about adding the feature highlights now.

### CODE REVIEW

File: src/hexo/themes/documentation/layout/partials/home.hbs [hunk 1]
```diff
@@ -9,16 +9,58 @@
...
+                                take advantage of all the features the web has to offer.
+                            </p>
+                        </div>
+                        <div class="module">
+                            <p class="subtitle--alt subtitle--home uppercase-text">Guidance tailored to your needs</p>
+                            <p class="feature_highlight">
+                                An online shopping experience probably doesn't have the same requirements as an intranet website.
+                                sonar is useful no matter what you are working on, and will provide you the best guidance for
+                                your needs. See how sonar can adapt its feedback when you give it more information about the resources
+                                you [cannot control](sectionNeedsAHeader), the <a href="/docs/user-guide/#browserconfiguration">browsers your visitors use</a>,
```
Comment (ststimac) [inline src/hexo/themes/documentation/layout/partials/home.hbs]: We need a hyperlink here @molant which is what I assume the (sectionsNeedsAHeader) is in reference to.

Comment (alrra) [reply src/hexo/themes/documentation/layout/partials/home.hbs]: @ststimac @molant https://sonarwhal.com/docs/user-guide/#ruleconfiguration ?

Comment (molant) [reply src/hexo/themes/documentation/layout/partials/home.hbs]: I've [created a new one](https://github.com/sonarwhal/sonar/commit/c2e2ca0a8313942e338c59358941dd5fa0c22da5), waiting for it to deploy

**Comment by molant** [comment]
> Home page looks much better IMO. 

Can you please post a screenshot?

**Comment by ststimac** [comment]
@molant  

screen shot 2017-09-12 at 1 59 26 pm

screen shot 2017-09-12 at 1 59 36 pm

**Comment by molant** [comment]
@ststimac I think we can remove the part where we say it requires node 8. We enforce it now via the package.json

File: src/hexo/themes/documentation/layout/partials/home.hbs (hunk 1 above)
Comment (alrra) [reply src/hexo/themes/documentation/layout/partials/home.hbs]: > waiting for it to deploy

[Done](https://sonarwhal.com/docs/user-guide/#ignoringdomains).

Comment (alrra) [reply src/hexo/themes/documentation/layout/partials/home.hbs]: I've fixed this before merging.

**Comment by alrra** [comment]
> remove the part where we say it requires node 8

I've fixed that before merging.

**Pull request closed without merging**